baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -12,6 +12,10 @@ Header name: `baggage`
 In order to increase interoperability across multiple protocols and encourage successful integration,
 implementations SHOULD keep the header name lowercase.
 
+## Header Encoding
+
+This header is a [[UTF-8]] encoded [[UNICODE]] string, however it uses only code points from the Basic Latin Unicode Block which are encoded identically in both Unicode and [[ASCII]].
```

Comment:
I feel this section makes it more ambiguous - for example, does this apply to only values?  Or does it apply to keys as well? I would suggest not having this new section and instead providing a link to the definition of code point in the place where we first reference it.

Reply:
+1. Good point. Let's move this sentence under the value section

Reply:
This section doesn't apply to just the value or the key, but to the whole header. The entire header is encoded as UTF-8 (indeed, all headers are). Lower down in `key` and `value` section we talk about which specific code points are allowed, but this section applies to the whole header so isn't included in those specific sections.